src/app/window-handler.ts

```diff
@@ -699,6 +704,20 @@ export class WindowHandler {
             });
         }
 
+        if (displayId !== '') {
+            const displays = electron.screen.getAllDisplays();
+
+            displays.forEach((element) => {
+                if (displayId === element.id.toString()) {
+                    this.createScrenSharingFrameWindow('screen-sharing-frame',
```

Comment:
@johankwarnmarksymphony JIRA also includes highlighting of the application but that's not working here

Reply:
@KiranNiranjan I don't understand what you mean.
I loop over all displays and if Id match I draw the frame around that display

Reply:
Displays is working but I mean highlighting the applications is not

Reply:
True, I will talk to the designer, if we should wait until we have fixed the highlighting of the application. Or split into two tickets 
1) Frame around sharing desktop
2) Frame around sharing application